saliency/xrai_test.py

```diff
@@ -0,0 +1,253 @@
+import logging
+import mock
+import numpy as np
+import skimage.draw as sk_draw
+import tensorflow as tf
+from tensorflow.python.platform import googletest
+from . import xrai
+from .xrai import XRAI, XRAIParameters
+
+IMAGE_SIZE = 299
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
> is it common practice to set logging level from within test scripts?

I am not sure about it. If a test fails, it is nice to have as much information as possible; especially if we had an automated daily test runs. Since the tests are run manually, it is a matter of preference.

Reply:
This is not common and might result in large logs making it harder to find where the failed test it. Can you remove this?